src/online/atc-info.class.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ATCInfo {
+  @ApiProperty({ description: 'The atc callsign', example: 'EBBR_TWR' })
+  callsign:string;
+
+  @ApiProperty({ description: 'The atc frequency', example: '128.800' })
+  frequency:string;
+
+  @ApiProperty({ description: 'The atc visual range', example: 150 })
+  visualRange:number;
+
+  @ApiProperty({ description: 'The atc current ATIS', example: ['line 1', 'line2', 'line3'] })
+  textAtis: string[];
+
+  @ApiProperty({ description: 'The atc type', example: 'GND' })
+  type:string;
```

Comment:
Should the type be a string or is an enum more favorable?

Reply:
Updated